core/src/commonTest/kotlin/arrow/endpoint/model/UriTest.kt

```diff
@@ -214,10 +214,12 @@ class UriTest : FunSpec() {
       parseToUri("http://\n/").shouldBeLeft().shouldBeTypeOf<UriError.InvalidHost>()
       parseToUri("http:// /").shouldBeLeft().shouldBeTypeOf<UriError.InvalidHost>()
       parseToUri("http://%20/").shouldBeLeft().shouldBeTypeOf<UriError.InvalidHost>()
-      parseToUri("http://abcd")
-        .fold({ fail("this should work") }, { UriCompatibility.encodeDNSHost(it.host().toString()) shouldBe "abcd" })
-      parseToUri("http://ABCD")
-        .fold({ fail("this should work") }, { UriCompatibility.encodeDNSHost(it.host().toString()) shouldBe "abcd" })
+      parseToUri("http://abcd").fold(
+        { fail("this should work") },
```

Comment:
This can all become `shoudBeRight()`, no?

Reply:
Correct, there are also other places where this fix applies, but I am focusing on getting the platform tests to compile first.